src/ui.h

```diff
@@ -505,7 +477,7 @@ class uilist // NOLINT(cata-xy)
         int vmax = 0;
 
         int desc_lines = 0;
-        int category_lines = 0;
+        // int category_lines = 0;
```

Comment:
remove?
```suggestion
```

Reply:
Technically there is one menu that tries to use categories. I’m not displaying them, but it does try to use them. Should the uilist support categories or should the spellcasting menu be its own thing? This is like menus with a second column, extra complexity that is used only in one place.

Reply:
That said, both `category_lines` and `desc_lines` are just temporary values, so there's no point in having them stored in the class anyway.